docs/content/docs/solana-vault-integration.mdx

```diff
@@ -16,30 +16,32 @@ The `VaultSDK` class provides a streamlined interface for interacting with Borin
 To use the Solana integration, you first need to create a connection to the Solana network and initialize the SDK:
 
 ```tsx
-import { Connection } from '@solana/web3.js';
+import { createSolanaClient } from 'gill';
 import { VaultSDK } from 'boring-vault-ui';
 
-// Initialize a connection to Solana
-const connection = new Connection('https://api.mainnet-beta.solana.com', 'confirmed');
+// Initialize a Solana client using gill
+const rpcUrl = 'https://api.mainnet-beta.solana.com';
```

Comment:
Do we want users to be able to initialize their own RPCs? 

Not necessary at this moment but a good point to have in mind.

cc @philipjames44